src/prometheus_server.py

```diff
@@ -30,8 +30,11 @@ def build_info(self):
         except urllib3.exceptions.MaxRetryError:
             return {}
 
-        info = json.loads(response.data.decode("utf-8"))
-        if info["status"] == "success":
-            return info["data"]
-        else:
+        try:
```

Comment:
This commit has been merged into master, with some modifications. You may want to rebase.